db/migrate/20150707062145_add_domain_to_client.rb

```diff
@@ -0,0 +1,5 @@
+class AddDomainToClient < ActiveRecord::Migration
+  def change
+    add_column :clients, :domain, :string
```

Comment:
This is more useful as a URL instead of just the domain. So this field should be renamed to `site_url`.